src/features/abi-methods/mappers/index.ts

```diff
@@ -20,3 +37,82 @@ export const jsonAsArc4AppSpec = (json: unknown): Arc4AppSpec => {
 
   return json as Arc4AppSpec
 }
+
+export const asAbiMethodArgumentRepresentation = (abiMethodArgument: AbiMethodArgument): AbiMethodArgumentRepresentation => {
+  if (
+    abiMethodArgument.type === AbiType.Address ||
+    abiMethodArgument.type === AbiType.Application ||
+    abiMethodArgument.type === AbiType.Account ||
+    abiMethodArgument.type === AbiType.Transaction ||
+    abiMethodArgument.type === AbiType.Asset
+  ) {
+    return {
+      ...abiMethodArgument,
+      multiLine: false,
+      length: `${abiMethodArgument.value}`.length,
+    }
+  }
+  const resolvedArgument = asAbiValueRepresentation(abiMethodArgument)
+  const multiLine = sum([resolvedArgument.length]) > MAX_LINE_LENGTH
+
+  return {
+    ...abiMethodArgument,
+    ...resolvedArgument,
+    multiLine: multiLine,
+  }
```

Comment:
maybe I am missing some context, but I have some questions about this section:
- sum of an array with just 1 element isn't needed.
- the returned value of `asAbiValueRepresentation` should indicate the flag `multiLine`, why do you need to re-calculate it here?